MNE/disp3D/engine/model/items/network/networktreeitem.cpp

```diff
@@ -238,7 +238,7 @@ void NetworkTreeItem::plotNetwork(const Network& tNetworkData, const QVector3D&
 
         //Plot out edges
         for(int j = 0; j < lNetworkNodes.at(i)->getEdgesOut().size(); ++j) {
-            if(abs(lNetworkNodes.at(i)->getEdgesOut().at(j)->getWeight()) >= vecThreshold.x()) {
+            if(fabs(lNetworkNodes.at(i)->getEdgesOut().at(j)->getWeight()) >= vecThreshold.x()) {
```

Comment:
Thx for the hint. fabs() without std:: should work fine here though. But for the sake of good practice, I changed it to use the namespace std:: and to prevent wrong abs() usage (ex. for floating values) in the future. I double checked some code and from what I saw fabs() or abs() are correctly used when dealing with floating or integers values. Please let me know if you find otherwise.